main.py

```diff
@@ -0,0 +1,23 @@
+
+p1_scr = 0
+p2_scr = 0
+deck = {'1', '2', '3', '4', '5', '6', '7', '8', '9', '10'}
+while len(deck) > 0:
```

Comment:
What is the difference between (hint: all of them work in python :) ) :
while deck 
while len(deck) 
while len(deck) > 0

Reply:
These are all similar boolean expressions.
As long as deck is not empty, we'll go into the while loop.
You're right, I could've written it in a shorter way :)